README.md

```diff
@@ -12,92 +12,33 @@ Fixd is a JavaScript utility for creating better fixtures. Fixtures created with
 ## Quick Start
 
 ```javascript
-// Require the library
+'use strict'
 const fixd = require('fixd')
 
-// Add a fixture
-fixd.addObject('foo', { foo: 'bar' })
+// Add a new fixture
+fixd.add('foo', { fizz: 'buzz', baz: { quz: 'baz' } })
 
-// Create a new instance of the fixture
-const fooOne = fixd.create('foo') // -> { foo: 'bar' }
+// Access fixture
+const foo = fixd.foo // -> { fizz: 'buzz', baz: { quz: 'baz' } }
 
-// Create a different instance with alterations to original object
-const fooTwo = fixd.create('foo', { foo: 'baz' }) // -> { foo: 'baz' }
+// Throws error
+foo.fizz = 'bazz' // -> ERROR: Cannot assign to read only property
```

Comment:
Since this is even suggested in the docs, can't we allow this as a one-time assignment?

```js
fixd.foo = {} // ok!
fixd.foo = {} // nope! Already set.
```

I suggest the same to @levithomason since it would offer parity w/ the getter, making the API more symmetrical aside from the `create` for mutations.